Resources/public/js/views/services/plugins/extensions/ez-draftserversidevalidation.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-draftserversidevalidation', function (Y) {
+    "use strict";
+    /**
+     * Provides the draft server side validation
+     *
+     * @module ez-draftserversidevalidation
+     */
+    Y.namespace('eZ');
+
+    /**
+     * Draft server side validation. It gives the shared methods between ez-publishdraftplugin and ez-savedraftplugin
```

Comment:
too many details that will eventually become wrong. You should write something like:
```
Draft server side validation. It provides the ability to extract the field validation errors from an error response.
```